app/components/primer/state_component.rb

```diff
@@ -4,12 +4,19 @@ module Primer
   # Component for rendering the status of an item.
   class StateComponent < Primer::Component
     COLOR_DEFAULT = :default
-    COLOR_MAPPINGS = {
+    NEW_COLOR_MAPPINGS = {
+      open: "State--open",
+      closed: "State--closed",
+      merged: "State--merged"
+    }.freeze
+
+    DEPRECATED_COLOR_MAPPINGS = {
       COLOR_DEFAULT => "",
-      :green => "State--green",
-      :red => "State--red",
-      :purple => "State--purple"
+      green: "State--open",
+      red: "State--closed",
+      purple: "State--merged"
```

Comment:
```suggestion
      :green => "State--open",
      :red => "State--closed",
      :purple => "State--merged"
```

linting strikes again